src/cowboy_http2.erl

```diff
@@ -69,6 +69,10 @@
 }.
 -export_type([opts/0]).
 
+%% Estimated time of a round-trip; the time to wait after initiating graceful
+%% shutdown before we stop accepting new streams.
+-define(shutdown_rtt, 500).
```

Comment:
It should be a configurable option. `shutdown_something_timeout`. I don't think rtt is descriptive enough.

Reply:
OK. `initial_shutdown_timeout` or `shutdown_initiation_timeout`?